graph/types/blockchain.go

```diff
@@ -26,6 +27,25 @@ func UnmarshalAddress(v interface{}) (common.Address, error) {
 	}
 }
 
+func MarshalBytes(b []byte) graphql.Marshaler {
+	return graphql.WriterFunc(func(w io.Writer) {
+		out, _ := json.Marshal(b)
```

Comment:
We can use `hexutil.Encode` to avoid incurring the JSON overhead. Also, when we're doing JSON stuff I'd like to keep using the one, faster package we're using elsewhere.